### PR TITLE
Feature/photo gallery

### PR DESCRIPTION
**Pull request by sivareddyuppathi**
Showcase your team with pride using our stunning Photo Gallery Component.

### CODE REVIEW

File: components/content/inspira/ui/photo-gallery/PhotoGallery.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  </div>
+</template>
+
+<script setup lang="ts">
+import { cn } from "~/lib/utils";
+
+interface Props {
+  containerClass?: string;
+  class?: string;
+  items: {
```
Comment (rahul-vashishtha) [inline components/content/inspira/ui/photo-gallery/PhotoGallery.vue:28]: Is it necessary to add a sub object to items? If it has only `src`, then I think `items` can be changed to `images: string[]`.
Can you clarify on that.

File: content/2.components/photo-gallery.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+::ComponentLoader{label="Preview" componentName="PhotoGalleryDemo" type="examples" id="photo-gallery"}
+::
+
+## API
+
+| Prop Name        | Type     | Default             | Description                                            |
+| ---------------- | -------- | ------------------- | ------------------------------------------------------ |
+| `items`          | `[]`     | `"[{src: string}]"` | Pass items / image src to animate                      |
```
Comment (rahul-vashishtha) [inline content/2.components/photo-gallery.md:16]: Seems like the type and default value information is swapped here.

**Review by rahul-vashishtha — CHANGES_REQUESTED** [review]

File: components/content/inspira/ui/photo-gallery/PhotoGallery.vue (hunk 1 above)
Comment (sivareddyuppathi) [reply components/content/inspira/ui/photo-gallery/PhotoGallery.vue:28]: In future If we want to add more fields related to each `item` for example `title`, no need to change the structure, so for future support added as Array of objects

File: content/2.components/photo-gallery.md (hunk 2 above)
Comment (sivareddyuppathi) [reply content/2.components/photo-gallery.md]: Done @rahul-vashishtha

**Review by rahul-vashishtha — APPROVED** [review]

**Pull request merged**